public/components/trace_analytics/components/services/service_view.tsx

```diff
@@ -301,28 +235,23 @@ export function ServiceView(props: ServiceViewProps) {
               <EuiFlexItem grow={true}>
                 {renderDatePicker(startTime, setStartTime, endTime, setEndTime)}
               </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiPopover
-                  panelPaddingSize="none"
-                  button={actionsButton}
-                  isOpen={actionsMenuPopover}
-                  closePopover={() => setActionsMenuPopover(false)}
-                >
-                  <EuiContextMenu initialPanelId={0} panels={actionsMenu} size="s" />
-                </EuiPopover>
-              </EuiFlexItem>
+              {serviceActionsMenu(_page === 'serviceFlyout')}
```

Comment:
this can be `page` rather than `_page`?

Reply:
I think this was done earlier to skip linting as parent also has the same name variable. Changed it to currentPage.